src/components/Menu/MenuItem.tsx

```diff
@@ -31,6 +41,9 @@ class MenuItem extends UIComponent<any, any> {
     /** Shorthand for primary content. */
     content: customPropTypes.contentShorthand,
 
+    /** Initial submenuOpened value. */
+    defaultSubmenuOpened: PropTypes.bool,
+
```

Comment:
This comment will be used in the docs page for describing the property. Please use more descriptive comment avoiding any properties.

Reply:
@mnajdova that makes sense. But I've made the same description as for ```defaultActiveindex``` prop in Accordion and Menu components to keep consistency. Should I change it?

Reply:
Aah, alright then, leave it as is.